docs/en/user_guides/1_config.md

```diff
@@ -226,14 +226,46 @@ load_from = None  # Load checkpoint from file.
 resume = False  # Whether to resume from existed model.
 ```
 
+These are all the configs for training and testing PSPNet, to load and parse them, we can use [Config](https://mmengine.readthedocs.io/en/latest/tutorials/config.html) implemented in [MMEngine](https://github.com/open-mmlab/mmengine)
+
+```python
+from mmengine.config import Config
+
+cfg = Config.fromfile('configs/pspnet/pspnet_r50-d8_4xb2-40k_cityscapes-512x1024.py')
+print(cfg.train_dataloader)
+```
+
+```shell
+{'batch_size': 2,
+ 'num_workers': 2,
+ 'persistent_workers': True,
+ 'sampler': {'type': 'InfiniteSampler', 'shuffle': True},
+ 'dataset': {'type': 'CityscapesDataset',
+  'data_root': 'data/cityscapes/',
+  'data_prefix': {'img_path': 'leftImg8bit/train',
+   'seg_map_path': 'gtFine/train'},
+  'pipeline': [{'type': 'LoadImageFromFile'},
+   {'type': 'LoadAnnotations'},
+   {'type': 'RandomResize',
+    'scale': (2048, 1024),
+    'ratio_range': (0.5, 2.0),
+    'keep_ratio': True},
+   {'type': 'RandomCrop', 'crop_size': (512, 1024), 'cat_max_ratio': 0.75},
+   {'type': 'RandomFlip', 'prob': 0.5},
+   {'type': 'PhotoMetricDistortion'},
+   {'type': 'PackSegInputs'}]}}
+```
+
+`cfg` is an instance of `mmengine.config.Config`, its interface is the same as a dict object and also allows access config values as attributes. See config tutorial in [MMEngine](https://github.com/open-mmlab/mmengine/blob/main/docs/zh_cn/tutorials/config.md) for more information.
```

Comment:
I suggest use English version, and move doc link to tutorial and add engine github link for  mmengine